src/faq_bot/config/paths.yaml

```
@@ -0,0 +1,7 @@
+paths:
+  base_dir: /Users/to-hashiguchi/workspace/faq_bot
+  data:
+    raw: ${paths.base_dir}/data/raw
+    processed: ${paths.base_dir}/data/processed
+    db: ${paths.data.processed}/faq.db
+    faiss_index: ${paths.data.processed}/faiss_index 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)